models/articles.models.js

```diff
@@ -3,9 +3,14 @@ const {
   articlesSortByCheck,
   articlesOrderByCheck,
 } = require("../db/seeds/utils");
-exports.fetchArticles = (sort_by = "created_at", order_by = "DESC") => {
+exports.fetchArticles = (
+  sort_by = "created_at",
+  order_by = "DESC",
+  topic = "topic"
+) => {
   const validSortBy = articlesSortByCheck(sort_by);
   const validOrderBy = articlesOrderByCheck(order_by);
+  let queryStr = `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count FROM comments LEFT JOIN articles ON comments.article_id = articles.article_id WHERE topic=${topic} GROUP BY articles.article_id ORDER BY ${sort_by} ${order_by}`;
```

Comment:
Using template literals for the WHERE clause opens you up to SQL injection! You can use `$1` here.